packages/trader/src/Assets/Trading/Categories/contract-type-description-video.tsx

```diff
@@ -11,13 +11,24 @@ type TContractTypeDescriptionVideo = {
 const ContractTypeDescriptionVideo = ({ selected_contract_type, data_testid }: TContractTypeDescriptionVideo) => {
     const { ui } = useStore();
     const { is_dark_mode_on: is_dark_theme } = ui;
+
+    let contract_name: string | undefined;
+    switch (selected_contract_type) {
```

Comment:
Add `switch-case` instead of `if-else` because in the nearest future there will be need to add video for `Turbos`, which also has 2 cases for one` contract_name`:
```
case TURBOS.LONG:
case URBOS.SHORT:
            contract_name = 'turbos';
            break;
```

Reply:
@kate-deriv it's better to remove this switch-case check from this file and pass the correct name as `selected_contract_type` prop in packages/trader/src/Assets/Trading/Categories/trade-categories-gif.tsx like I suggested here:
https://github.com/binary-com/deriv-app/pull/9858/files#r1311004548

Reply:
Thank you, will do